src/Utils/obiDialogParser.ts

```diff
@@ -21,11 +21,21 @@ enum OBIRuleType {
     INTENT_RULE = "Microsoft.IntentRule"
 }
 
+export interface ObiDialogParserResult {
+    luMap: Map<string, string[]>
```

Comment:
You seemed to be avoiding string maps in other PRs but chose to use here even though key is still `string`.
What was reason for this one?

Reply:
Well given it's merged already suppose it doesn't matter. But still interested in why they are treated differently

Reply:
My understanding was that we agreed on doing post-merge changes for stuff the is time sensitive.    In any case, in answer to your comment, laziness. :)   I've been gradually switching stuff over and was editing the lg part.  LU not really used yet, but I'll make that change too.

Reply:
Hmm. I suppose I suspected this but wanted to believe otherwise.  Makes me not want to do code reviews if comments are skipped out of laziness. 

> I've been gradually switching stuff over 

This is just more laziness. It's easier for you to makes changes where you're currently working instead of everywhere but it's intentionally introducing inconsistency or pattern that contradicts the old. You lowered your cost of changes cost was then multiplied by everyone else who sees it.  Just leads to confusion. We wouldn't be having this conversation if it wasn't there so now it's taking more time.